### DISCUSSION TITLE
可查询的人物信息要不要加入旅行者

### DISCUSSION THREAD

**Discussion by mark9804** [Arondight/Adachi-BOT#327] [Ideas]
虽然不太能理解，但是群里很多人想查旅行者的信息

因为旅行者有几个不同的属性，所以如果要做的话就要做成风主、岩主、雷主三个json，如果查询`空`或者`荧`的话默认返回风主

**Comment by Arondight** [comment]
工作量实在是太大了，因为声优、素材、技能的不一致，需要做 7x2 个角色，有了风就有人要查岩，还有人要查水火冰
